classes/shop/Shop.php

```diff
@@ -464,13 +465,36 @@ public function getAddress()
     }
 
     /**
-     * Get shop theme name
-     *
-     * @return string
+     * Set shop theme details from Json data
      */
+    public function setTheme()
+    {
+        $dir = _PS_ALL_THEMES_DIR_.$this->theme_name;
+        $theme_data = Yaml::parse(file_get_contents(
```

Comment:
Hum... how about use Config component and use YamLoader ? This lets us a chance to provide more ways to create theme configurations later, for instance XML loader, CSV loader ...